models/google_ads.yml

```diff
@@ -40,22 +48,176 @@ models:
+          combination_of_columns:
+            - ad_id
+            - ad_group_id
+            - date_day
+    columns:
+      - name: date_day
+        description: "{{ doc('date') }}"
+      - name: account_name
+        description: "{{ doc('account_descriptive_name') }}"
+      - name: account_id
+        description: "{{ doc('external_customer_id') }}"
+      - name: campaign_name
+        description: "{{ doc('campaign_name') }}"
+      - name: campaign_id
+        description: "{{ doc('campaign_id') }}"
+      - name: ad_group_name
+        description: "{{ doc('ad_group_name') }}"
+      - name: ad_group_id
+        description: "{{ doc('ad_group_id') }}"
+      - name: ad_id
+        description: The unique identifier of the ad.
+      - name: display_url
+        description: The url which is displayed with the ad.
+      - name: source_final_urls
+        description: The final urls that are used within the ad.
+      - name: ad_status
+        description: The status of the ad.
+      - name: ad_type
+        description: The type of the ad that is being served.
+      - name: spend
+        description: "{{ doc('cost') }}"
+      - name: clicks
+        description: "{{ doc('clicks') }}"
+      - name: impressions
+        description: "{{ doc('impressions') }}"
+
+  - name: google_ads__campaign_report
+    description: Each record in this table represents the daily performance of a campaign at the campaign/advertising_channel/advertising_channel_subtype level.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - campaign_id
+            - advertising_channel_type
+            - advertising_channel_subtype
+            - date_day
+    columns:
+      - name: date_day
+        description: "{{ doc('date') }}"
+      - name: account_name
+        description: "{{ doc('account_descriptive_name') }}"
+      - name: account_id
+        description: "{{ doc('external_customer_id') }}"
+      - name: campaign_name
+        description: "{{ doc('campaign_name') }}"
+      - name: campaign_id
+        description: "{{ doc('campaign_id') }}"
+      - name: advertising_channel_type
+        description: The channel type of the ads being served within the campaign.
+      - name: advertising_channel_subtype
+        description: The channel subtype of the ads being served within the campaign.
+      - name: status
+        description: The status of the campaign.
+      - name: spend
+        description: "{{ doc('cost') }}"
+      - name: clicks
+        description: "{{ doc('clicks') }}"
+      - name: impressions
+        description: "{{ doc('impressions') }}"
+
+  - name: google_ads__keyword_report
+    description: Each record in this table represents the daily performance at the ad group level for keywords.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - ad_group_id
+            - criterion_id
```

Comment:
Is this a unique identifier for keyword_text? e.g. for ad_group level unique combos, we could use ad_group_id in place of ad_group_name, would this be the same logic?

Reply:
Not sure I am following this one 🤔

Reply:
Your explanation makes sense! Resolving

Reply:
Based off our slack convo I think we can keep it as it is.